src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -425,7 +425,7 @@ enum UlduarData
     DATA_HODIR_YS,
     DATA_THORIM_YS,
     DATA_MIMIRON_YS,
-    DATA_ILLUSION, // = 38 used by conditions
```

Comment:
It was moved to cpp a long time ago. And tbh it wasn't even in the scope of the PR